app/vmalert/main.go

```diff
@@ -91,7 +91,12 @@ absolute path to all .tpl files in root.
 
 	disableAlertGroupLabel = flag.Bool("disableAlertgroupLabel", false, "Whether to disable adding group's Name as label to generated alerts and time series.")
 
-	dryRun = flag.Bool("dryRun", false, "Whether to check only config files without running vmalert. The rules file are validated. The -rule flag must be specified.")
+	dryRun        = flag.Bool("dryRun", false, "Whether to check only config files without running vmalert. The rules file are validated. The -rule flag must be specified.")
+	unitTest      = flag.Bool("unittest", false, "Used to perform unit testing with given unit testing files")
```

Comment:
would be nice to add a chapter to the README with explanation what this new mode is doing and how to use it.

Reply:
Right! Added, please check)